app/my_script.py

```diff
@@ -1,4 +1,118 @@
-import os
-import pwd
+import sys
+import json
+import logging
+import pandas as pd
 
-print("Hello {} - welcome to my template-world!".format(pwd.getpwuid(os.getuid())[0]))
+# Initialize log
+log = logging.getLogger(__name__)
+
+DLR_MRID_FILEPATH = '/home/tsp/github/conduck/tests/valid-testdata/DLR_MRID.csv'
+MRID_KEY_NAME = 'terminal_name'
+MRID_VALUE_NAME = 'segment_id'
+MRID_EXPECTED_HEADER_NAMES = ['terminal_name', 'far_near', 'measurement_id', 'segment_id', 'dlr_enable']
+
+LIMITS_NAME_FILEPATH = '/home/tsp/github/conduck/tests/valid-testdata/Limits_other.xlsx'
+DD20_DATASHEET_NAME = 'DD20Mapping'
+DD20_KEY_NAME = 'DD20 Name'
+DD20_VALUE_NAME = 'ETS Name'
+
+
+def parse_csv_file_to_dataframe(file_path: str, header_index: int = 0) -> pd.DataFrame:
+    '''
+    Read CSV file and parse it to pandas dataframe.
+    Parameters
+    ----------
+    file_path: str
+        Full path of the excel file.
+    header_index: int
+        Index number for row to be used as header (Default = 0)
+    Returns
+    -------
+        pd.DataFrame
+            A dataframe containing the data from csv file.
+    '''
+    # Trying to read data from CSV file and convert it to a dataframe.
+    try:
+        dataframe = pd.read_csv(file_path, delim_whitespace=True, on_bad_lines='skip',
+                                header=header_index)
+        log.info(f'CSV data in "{file_path}" was parsed to dataframe.')
+    except Exception as e:
+        log.exception(f'Parsing data from: "{file_path}" failed with message: {e}.')
+        sys.exit(1)
```

Comment:
I would suggest using 'raise' in stead of manually exiting. That way you don't have to import the sys module for just exiting, and the 'raise' will call a system exit anyway if unhandled. But the 'clever' thing is that i can actually be handled higher up in the stack. That way you are using the built-in error/exception system and your function will survive being called within a 'try/except' in case someone actually wants to handle any errors "higher up".

```suggestion
        raise
```
or 
````suggestion
        raise IOError("File not found or incorrectly formatted") from e
````

The above suggestions are examples of how it COULD be done - ONE of them should be selected, or a similar approach. The first suggestion will just re-raise the exception that happened - the second suggestion will add another exception to the stack (giving more information, generally).

Reply:
Thanks for making me aware of the in built raise. It does the same and more. :)
It is included to code.